tailwind.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
       'primary-dark': 'rgb(0,88,219)',
       red: 'rgb(255,13,0)',
       'red-dark': 'rgb(230,11,0)',
-      orange: 'rgb(255,149,0)',
+      orange: 'rgb(255,149,0,1)',
```

Comment:
Be careful with modifying the color in the tailwind config because you change the color for the whole project. If you only need to change it for a specific case, you can try with [arbitrary values](https://tailwindcss.com/docs/background-color#arbitrary-values).